spec/services/brewery_search_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe BrewerySearch do
+  describe ".find_brewery" do
+    it "returns the first brewery with the name sent in the search parameter" do
+      brewery_name = "Anchor"
+
+      stub_request(:any, "https://brianiswu-open-brewery-db-v1.p.rapidapi.com/breweries/search?query=#{brewery_name}").
+        with(
+          headers: {
+            "Host" => "brianiswu-open-brewery-db-v1.p.rapidapi.com:443",
+            "X-Rapidapi-Host" =>
+            "brianiswu-open-brewery-db-v1.p.rapidapi.com",
+            "X-Rapidapi-Key" =>
+            "d06411ee49mshcbcdc80cdd60989p1f0313jsn80c1ad5cec44",
+          },
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.`